src/index.ts

```diff
@@ -7,18 +7,48 @@ import {
 } from './config'
 import { fromQuery, fromUntilQuery, untilQuery } from './media-queries'
 
-type Until = { [key in Breakpoints]: string }
+/*
+`from` and `until` are objects with overridden `toString` methods,
+so that they can be coerced into media query strings _or_ provide
+further refinement of the query. this freaky genius was thought up
+by @SiAdcock.
+
+e.g.:
+
+const styles = {
+	[from.small]: {
+		color: 'red'
+	},
+	[from.small.until.large]: {
+		color: 'blue'
+	},
+	[from.small.until.large.for('print')]: {
+		color: 'black'
+	},
+}
+*/
+
+type Until = {
+	[key in Breakpoints]: {
+		toString: () => string
+		for: (arg0: string) => string
```

Comment:
Interesting! Why not give the argument a more meaningful name (e.g. `mediaType`)?